### ISSUE TITLE
[Docs]: 

### ISSUE THREAD

**Issue by dasanirban834** [hashicorp/terraform-provider-aws#29850]
### Documentation Link

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb_target_group_attachment

### Description

Hi Team,

I found one loopholes in aws_lb_target_group_attachment documentation, where target id defines instance id, but during launching new aws instance using resource block , we can't get id attribute as an output. Then how can we put target id in target attachment ? Please find below link -->

1. aws_instance: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/instance#attributes-reference

2. aws_lb_target_group_attachment: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb_target_group_attachment

### Would you like to implement a fix?

None

**Comment by justinretzolk** [comment]
Hey @dasanirban834 👋 Thank you for taking the time to raise this! In order to get the ID of an AWS instance created with the `aws_instance` resource, you can use `aws_instance.id`. Am I correct that that's what you were looking for?

**Comment by danivideda** [comment]
> Hey @dasanirban834 👋 Thank you for taking the time to raise this! In order to get the ID of an AWS instance created with the `aws_instance` resource, you can use `aws_instance.id`. Am I correct that that's what you were looking for?

Yes it's exactly this. However, I'd also like to raise an issue with the docs. As I don't see any `id` reference in **Attribute Reference** section.
